include/flatbuffers/idl.h

```diff
@@ -1045,7 +1049,13 @@ class Parser : public ParserState {
   uint64_t advanced_features_;
 
  private:
+
+  enum ModuleAnnotation { kNone, kPublic, kPrivate };
+
   const char *source_;
+  // Add enum here for file type
+
+  ModuleAnnotation module_annotation_;
```

Comment:
Can you add a comment explaining what this field is used for?

Reply:
This field was removed with the new logic